2.cellprofiler_processing/README.md

```diff
@@ -8,16 +8,11 @@ To run segmentation, we use the standard approach used in CellProfiler for Cell
 
 This standard approach is as follows:
 
-- **IdentifyPrimaryObjects:** Identify nuclei from the `d0` channel images. This creates a group for an object called `OrigNuclei`, which includes nuclei that are a part of cells that touch an edge of the image.
+- **IdentifyPrimaryObjects:** Identify nuclei from the `d0` channel images (discarding ones that touch the edges of the image). This creates a group for an object called `Nuclei`.
 
-- **IdentifySecondaryObjects:** Identify whole cells using the nuclei from the previous module as a base from the `d4` channel (stained for F-actin) images. 
-These whole cells make up the object group called `Cells`.
-Any cells that are touching at least one edge of the image will be removed along with its respective nucleus. 
-This creates a new object group call `Nuclei`.
+- **IdentifySecondaryObjects:** Identify whole cells using the nuclei from the previous module as a base from the `d4` channel (stained for F-actin) images (discarding edge cells). These whole cells make up the object group called `Cells`.
 
-- **IdentifyTeritaryObjects:** Identify cytoplasm by subtracting out the "smaller identified objects" (`OrigNuclei`) from the "larger identified objects" (`Cells`). 
-`OrigNuclei` has to be used in this module over `Nuclei` due to an error downstream in the pipeline when using the latter object where it states "Cytoplasm is not in a 1:1 relationship with other objects".
-This means that the number of cytoplasm is not the same as the other objects, which would be wrong because there should be the same amount of objects (e.g the number of cells should match the number of cytoplasm and nuclei).
+- **IdentifyTeritaryObjects:** Identify cytoplasm by subtracting out the "smaller identified objects" (`Nuclei`) from the "larger identified objects" (`Cells`). 
```

Comment:
Are the quoted sections of this sentence references from elsewhere? If so, consider providing that context here with a link, paper reference, etc. If not, consider relying on using compartment names instead to assist with readability.

Reply:
In this case, they are referenced above in the previous sentences for the first and second modules for segmentation. I don't have any documentation like papers, and these names are the typical compartment names just without the prefix. I don't think it makes sense here to put the compartment names (e.g. `Per_Nuclei`, etc.).